src/main/java/mate/academy/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -29,8 +29,12 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(errorDto, headers, status);
     }
 
-    @ExceptionHandler({NotUniqueValueException.class, EntityNotFoundException.class})
-    public ResponseEntity<Object> handleCustomException(NotUniqueValueException ex) {
+    @ExceptionHandler({
+            NotUniqueValueException.class,
+            EntityNotFoundException.class,

```

Comment:
can we make another method for `EntityNotFoundEx` to return status code `404 - not found` instead of a bad request?